imp: recebe número da PR como input (#8)
betrybe/eslint-linter-action: README.md

A GitHub action that evaluates projects with [ESLint](https://eslint.org/) and comments the evaluation outcome on the student's pull request.

## Development

> **P.S.:** Github actions will run `dist/index.js` to execute this action, so you must to run `npm run pack`if you want apply any changes.

Reviewer: ```suggestion
⚠️ Github actions will run `dist/index.js` to execute this action, so you *must run* `npm run pack` and commit the changes made at `dist/index.js` if you want to apply any changes. ⚠️
```
